eng/native/build-commons.sh

```diff
@@ -48,11 +48,6 @@ check_prereqs()
 {
     echo "Checking prerequisites..."
 
-    if ! cmake --help 2>&1 | grep -q \\-B; then
-        echo "Please install cmake v3.14.5 or newer from https://www.cmake.org/download/."
```

Comment:
I think it'd still be good to keep this check since it makes the build experience much nicer when you're missing cmake (assuming the old cmake returns some parsable version number from `--help`).

Reply:
This check was also being hit by the RedHat build, so either we'd need to branch this check for Linux/OSX, or since we aren't using the `-B` and `-S` flags any more, we can just let CMake do the version check when it starts configuring.